src/basic/time-util.c

```diff
@@ -304,9 +303,28 @@ static char *format_timestamp_internal(
         struct tm tm;
         time_t sec;
         size_t n;
+        bool utc = false;
+        bool us = false;
 
         assert(buf);
 
+        switch(style) {
```

Comment:
doesn't matter, but we usually put a space between `switch` and `(`

Reply:
fixed